Library/Formula/fuseki.rb

```diff
@@ -22,67 +21,33 @@ def install
 
     # Install and symlink wrapper binaries into place
     libexec.install 'fuseki-server'
-    bins = ['s-delete', 's-get', 's-head', 's-post', 's-put', 's-query', 's-update', 's-update-form']
+    bins = ['bin/s-delete', 'bin/s-get', 'bin/s-head', 'bin/s-post', 'bin/s-put', 'bin/s-query', 'bin/s-update', 'bin/s-update-form', 'bin/soh']
     chmod 0755, bins
     libexec.install bins
     bin.install_symlink Dir["#{libexec}/*"]
     # Non-symlinked binaries and application files
-    libexec.install 'fuseki-server.jar', 'pages'
-
-    etc.install 'config.ttl' => 'fuseki.ttl'
+    libexec.install 'fuseki-server.jar', 'webapp'
 
     # Create a location for dataset and log files,
     # in case we're being used in LaunchAgent mode
     (var/'fuseki').mkpath
     (var/'log/fuseki').mkpath
 
-    # Install example configs
-    prefix.install 'config-examples.ttl', 'config-inf-tdb.ttl', 'config-tdb-text.ttl', 'config-tdb.ttl'
-
-    # Install example data
-    prefix.install 'Data'
-
     # Install documentation
-    prefix.install 'LICENSE', 'NOTICE', 'ReleaseNotes.txt'
+    prefix.install 'LICENSE', 'NOTICE'
   end
 
   def caveats; <<-EOS.undent
     Quick-start guide:
 
-    * See the Fuseki documentation for instructions on using an in-memory database:
-      http://jena.apache.org/documentation/serving_data/#fuseki-server-starting-with-an-empty-dataset
+    * See the Fuseki documentation for instructions on running Fuseki from the command line:
+      https://jena.apache.org/documentation/fuseki2/fuseki-run.html#fuseki-as-a-standalone-server
 
-    * Running from the LaunchAgent is different the standard configuration and
-      uses traditional Unix paths: please inspect the settings here first:
-      #{etc}/fuseki.ttl
+    * This version does not include the LaunchAgent version anymore!
 
-    * NOTE: Fuseki uses 1) log4j.configuration if defined, 2) 'log4j.properties' file if exists,
-      or built-in configuration. See also:
-      https://issues.apache.org/jira/browse/JENA-536
-    EOS
-  end
+    * You may want to set the FUSEKI_BASE environment variable which points to the runtime area for the server instance (defaults to 'run/' of the current directory).
```

Comment:
We could write a wrapper Bash script which sets the `FUSEKI_BASE` environment variable for the install.